ComponentThemeLiveReloadIT fails in Flow validation

ComponentThemeLiveReloadIT.webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly has been failing since that change 
https://github.com/vaadin/flow/commit/af4060779ae227fe9dbb9ace74b8aa60833f5c8e

For some reason the initial page is not loaded and the very first element check throws.

However, this test passes in the Flow PR validation and the manual component styles change works well.

Tried to add more  `waitFor` clauses and reorder this test to be the last one - it didn't solve the problem.

More details:

```
java.lang.RuntimeException: webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly[ANY_Chrome_](com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT): run failed 2 times
	at com.vaadin.testbench.RetryRule$1.evaluate(RetryRule.java:78)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:63)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.apache.maven.surefire.junitcore.pc.Scheduler$1.run(Scheduler.java:410)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.openqa.selenium.TimeoutException: Expected condition failed: waiting for presence of element located by: By.id: themed-component-id (tried for 10 second(s) with 500 milliseconds interval)
	at org.openqa.selenium.support.ui.WebDriverWait.timeoutException(WebDriverWait.java:95)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:272)
	at com.vaadin.testbench.TestBenchTestCase.waitUntil(TestBenchTestCase.java:220)
	at com.vaadin.testbench.TestBenchTestCase.waitUntil(TestBenchTestCase.java:244)
	at com.vaadin.flow.testutil.TestBenchHelpers.waitForElementPresent(TestBenchHelpers.java:136)
	at com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT.isComponentCustomStyle(ComponentThemeLiveReloadIT.java:164)
	at com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT.webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly(ComponentThemeLiveReloadIT.java:109)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at com.vaadin.testbench.parallel.ParallelRunner$TBMethod.invokeExplosively(ParallelRunner.java:490)
	at org.junit.internal.runners.statements.InvokeMethod.evaluate(InvokeMethod.java:17)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:26)
	at com.vaadin.testbench.parallel.ParallelRunner$1.evaluate(ParallelRunner.java:464)
	at org.junit.internal.runners.statements.RunAfters.evaluate(RunAfters.java:27)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at org.junit.rules.TestWatcher$1.evaluate(TestWatcher.java:61)
	at com.vaadin.testbench.RetryRule$1.evaluate(RetryRule.java:68)
	... 12 more
Caused by: org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"css selector","selector":"#themed\-component\-id"}
  (Session info: headless chrome=93.0.4577.63)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'ip-172-16-1-153.eu-west-1.compute.internal', ip: '172.16.1.153', os.name: 'Linux', os.arch: 'amd64', os.version: '4.14.146-119.123.amzn2.x86_64', java.version: '1.8.0_201'
Driver info: org.openqa.selenium.remote.RemoteWebDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 93.0.4577.63, chrome: {chromedriverVersion: 93.0.4577.63 (ff5c0da2ec0ad..., userDataDir: /tmp/.com.google.Chrome.oZ9cN6}, goog:chromeOptions: {debuggerAddress: localhost:43961}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: LINUX, platformName: LINUX, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:credBlob: true, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true, webdriver.remote.sessionid: b1348ab253a258969cccc3d666f...}
Session ID: b1348ab253a258969cccc3d666ff4f8b
*** Element info: {Using=id, value=themed-component-id}
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
	at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
	at org.openqa.selenium.remote.RemoteWebDriver.findElementById(RemoteWebDriver.java:372)
	at org.openqa.selenium.By$ById.findElement(By.java:188)
	at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
	at com.vaadin.testbench.TestBenchDriverProxy.findElement(TestBenchDriverProxy.java:51)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvst2d6_0._d6findElement(TestBenchDriverProxy_$$_jvst2d6_0.java)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at com.vaadin.testbench.DriverInvocationHandler.invoke(DriverInvocationHandler.java:48)
	at com.vaadin.testbench.TestBenchDriverProxy_$$_jvst2d6_0.findElement(TestBenchDriverProxy_$$_jvst2d6_0.java)
	at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:182)
	at org.openqa.selenium.support.ui.ExpectedConditions$6.apply(ExpectedConditions.java:179)
	at org.openqa.selenium.support.ui.FluentWait.until(FluentWait.java:249)
	... 32 more
------- Stderr: -------
Sep 24, 2021 9:31:14 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
Sep 24, 2021 9:32:55 AM com.vaadin.testbench.ScreenshotOnFailureRule failed
INFO: Error screenshot written to: /opt/agent/work/67776874a33a15db/flow-tests/test-application-theme/test-theme-component-live-reload/error-screenshots/webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly[ANY_Chrome_](com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT).png
Sep 24, 2021 9:32:55 AM org.openqa.selenium.remote.ProtocolHandshake createSession
INFO: Detected dialect: W3C
Sep 24, 2021 9:34:15 AM com.vaadin.testbench.ScreenshotOnFailureRule failed
INFO: Error screenshot written to: /opt/agent/work/67776874a33a15db/flow-tests/test-application-theme/test-theme-component-live-reload/error-screenshots/webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly[ANY_Chrome_](com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT).png
```

Err screenshot:

![image](https://user-images.githubusercontent.com/61410877/134657365-c78f69ed-bcaf-4242-99ed-562cba8a1996.png)

Currently, on master, the test fails with a different error

```
StylesCreatedAndDeleted_stylesUpdatedOnFly[ANY_Chrome_](com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT).png
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 3.706 s <<< FAILURE! - in com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT
[ERROR] webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly[ANY_Chrome_](com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT)  Time elapsed: 3.706 s  <<< ERROR!
org.openqa.selenium.NoSuchElementException: 
No element with tag <div> found with the attributes [class='vaadin-text-field-container'] using index 0
For documentation on this error, please visit: https://selenium.dev/exceptions/#no_such_element
Build info: version: '4.1.1', revision: 'e8fcc2cecf'
System info: host: 'marco-vivobook', ip: '127.0.1.1', os.name: 'Linux', os.arch: 'amd64', os.version: '5.13.0-27-generic', java.version: '11.0.13'
Driver info: driver.version: unknown
	at com.vaadin.testbench.ElementQuery.get(ElementQuery.java:318)
	at com.vaadin.testbench.ElementQuery.first(ElementQuery.java:242)
	at com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT.isComponentCustomStyle(ComponentThemeLiveReloadIT.java:169)
	at com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadIT.webpackLiveReload_newComponentStylesCreatedAndDeleted_stylesUpdatedOnFly(ComponentThemeLiveReloadIT.java:110)
```

This failure seems to be related to a change on `vaadin-text-field` internal structure

https://github.com/vaadin/web-components/issues/2220
https://github.com/vaadin/web-components/commit/2e2ff00af2410276abea829d6bb30fe647b8fad9#diff-a83912b755f5c355794ad6d79697531de9ce4704f2940d82febfe86b9f9b5c3aL84-L111

Once adapted `isComponentCustomStyle()` method to the new structure, the test always pass locally.

--- a/flow-tests/test-application-theme/test-theme-component-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ComponentThemeLiveReloadIT.java
+++ b/flow-tests/test-application-theme/test-theme-component-live-reload/src/test/java/com/vaadin/flow/uitest/ui/ComponentThemeLiveReloadIT.java
@@ -41,7 +41,6 @@ import com.vaadin.testbench.TestBenchElement;
 import static com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadView.ATTACH_IDENTIFIER;
 import static com.vaadin.flow.uitest.ui.ComponentThemeLiveReloadView.THEMED_COMPONENT_ID;
 
-@Ignore("Temporary solution for https://github.com/vaadin/flow/issues/11928")
 @NotThreadSafe
 public class ComponentThemeLiveReloadIT extends ChromeBrowserTest {
 
@@ -166,9 +165,9 @@ public class ComponentThemeLiveReloadIT extends ChromeBrowserTest {
             TestBenchElement themedTextField = $(TestBenchElement.class)
                     .id(THEMED_COMPONENT_ID);
             TestBenchElement input = themedTextField.$(DivElement.class)
-                    .attribute("class", "vaadin-text-field-container").first()
-                    .$(DivElement.class).attribute("part", "input-field")
-                    .first();
+                    .attribute("class", "vaadin-field-container").first()
+                    .$("vaadin-input-container")
+                    .attribute("part", "input-field").first();
             return borderRadius.equals(input.getCssValue("border-radius"));
         } catch (StaleElementReferenceException e) {
             return false;

